packages/calypso-e2e/src/data-helper.ts

```diff
@@ -34,16 +34,39 @@ export function getCalypsoURL(
 /**
  * Returns the credential for a specified account from the configuration file.
  *
- * @param {string} username Username of the account for which the password is to be obtained.
- * @returns {string} Credential entry found in the configuration file for the given username.
+ * @param {string} username Username of the account for which the credentials are to be obtained.
+ * @returns {string[]} Credential entry found in the configuration file for the given username.
  * @throws {Error} If username does not correspond to a valid entry in the configuration file.
  */
-export function getAccountCredential( username: string ): string {
+export function getAccountCredential( username: string ): string[] {
```

Comment:
I have changed  this method to only return the `username`/`password` as an array of strings.